packages/charts/src/charts/funnel/funnel-chart.component.tsx

```diff
@@ -23,7 +27,7 @@ export type Props = {
   /** Key used to pick value property from data */
   valueKey?: string;
   /** Function used to format step values */
-  formatValues?: (value: number) => string | number;
+  formatValues?: Formatter;
```

Comment:
Could You please, add unit tests that covers formatting value by using `string` pattern for funnel steps ?

Reply:
Test added